etc/rtp_hera_test1.cfg

```diff
@@ -32,25 +32,32 @@ prioritize_obs = 1
 ###
 neighbors = 0
 ###
+# pol_neighbors : 0 or 1, by default this is disabled, set = 1 to enable
+# This causes similar behavior to neighbors, but for polarization instead of in time
+###
+pol_neighbors = 1
+###
 # lock_all_neighbors_to_same_still : 0 or 1, if enabled then all neighbor obsid's will be routed to the same still server
 ###
-lock_all_neighbors_to_same_still = 0
+lock_all_neighbors_to_same_still = 1
 ###
 # actions : comma seperated list of actions this workflow will perform in exact order.
 # Do not use keywords NEW or COMPLETE, the system takes care of these internally, nothing is run on
 # either the NEW or COMPLETE state, they are for internal tracking.  Once a workflow is completely
 # finished it will be left in state COMPLETE.  To trigger the workflow set its Observation status to
 # the first action in the workflow, note that the first does not do anything except act as a placeholder
 ###
-actions=UV_POT,UV,UVC,CLEAN_UV,PLOTAUTOS,ADD_LIBRARIAN_PLOTAUTOS,CLEAN_PLOTAUTOS,ANT_FLAG,ADD_LIBRARIAN_BADANTS,PULL_SUBARRAY,ADD_LIBRARIAN_SUBARRAY,FIRST_CAL_HEX,ADD_LIBRARIAN_FIRSTCAL,CLEAN_FIRST_CAL_HEX,CLEAN_SUBARRAY,CLEAN_ANT_FLAG,CLEAN_UVC,DELETE_RAW,COMPLETE
+actions=UV_POT,UV,UVC,CLEAN_UV,PLOTAUTOS,CLEAN_PLOTAUTOS,PULL_SUBARRAY,CLEAN_UVC,ANT_METRICS,FIRSTCAL,OMNICAL,OMNI_APPLY,XRFI,COMPLETE
```

Comment:
Ideally with the cm system functioning correctly we will not need the UVC step.  Can we remove it?

Steps to be added:
ant_metrics_MC, antmetric_librarian, firstcal_metrics, firstcal_metrics_MC,firstcal_metrics_librarian, ibid omnical metrics, plot_rfi, rfi_librarian,clean_metrics?